tests/functional/externalpages/pocket/test_navigation.py

```diff
@@ -0,0 +1,37 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.webdriver.common.by import By
+
+from pages.externalpages.pocket.about import AboutPage
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_mobile_menu(base_url, selenium_mobile):
+    page = AboutPage(selenium_mobile, base_url).open()
+    assert page.navigation.is_mobile_menu_open_button_displayed
+    page.navigation.open_mobile_menu()
+    assert page.navigation.is_mobile_menu_home_link_displayed
+    assert page.navigation.is_mobile_menu_my_list_link_displayed
+
+    assert page.navigation.is_mobile_menu_close_button_displayed
+    page.navigation.close_mobile_menu()
+    assert not page.navigation.is_mobile_menu_home_link_displayed
+    assert not page.navigation.is_mobile_menu_my_list_link_displayed
+
+
+def test_accessible_mobile_menu_open_name(base_url, selenium_mobile):
+    expected_accessible_name = "Open"
+    page = AboutPage(selenium_mobile, base_url).open()
+    button_label_reference = page.navigation.mobile_menu_open_button.get_attribute("aria-labelledby")
+    assert page.navigation.mobile_menu_open_button.find_element(By.ID, button_label_reference).text == expected_accessible_name
+
+
+def test_accessible_mobile_menu_close_name(base_url, selenium_mobile):
+    expected_accessible_name = "Close the Pocket Mobile Menu"
+    page = AboutPage(selenium_mobile, base_url).open()
+    page.navigation.open_mobile_menu()
+    assert page.navigation.mobile_menu_close_button.text == expected_accessible_name
```

Comment:
This `assert` fails for me locally?

Whilst testing a11y is a great idea, generally we try and avoid testing against actual page copy (since text is often localized, and can also change without functionality changing). I'm not opposed to including including this for now, since the pocket pages aren't localized. But tests like this can tend to be a bit brittle over time.

Reply:
simplified to check for a non-empty string, not specific text